### ISSUE TITLE
Difference about training schedule between technical report and DiffSynth

### ISSUE THREAD

**Issue by RedRAINXXXX** [Wan-Video/Wan2.1#293]
Hi~ I noticed that the training schedule of the wanx T2V model is slightly different in the technical report and the [Diffsynth](https://github.com/modelscope/DiffSynth-Studio) repository. Specifically:

- The description in the official technical report uses logit-normal sampling for timesteps

- The implementation in the Diffsynth repository is [uniform sampling](https://github.com/modelscope/DiffSynth-Studio/blob/803ddcccc789a6f1da7c353ece631b4eae90a999/diffsynth/schedulers/flow_match.py#L22), timestep shift, and [loss weighting](https://github.com/modelscope/DiffSynth-Studio/blob/803ddcccc789a6f1da7c353ece631b4eae90a999/diffsynth/schedulers/flow_match.py#L32)

Which method did you use in training?

**Comment by GFENGG** [comment]
> [@RedRAINXXXX](https://github.com/RedRAINXXXX) Hello, I am a developer of DiffSynth-Studio.
> 
> Both of these approaches can be used for training. These differences also exist in the open-source training code of the FLUX model.
> 
> Our experimental results on FLUX indicate that such parameters perform the best in FLUX's LoRA training. Given the similarities between Wan and FLUX, we have adopted FLUX's experience in Wan's training.
> 
> We have already validated this approach in Wan's LoRA training, so you can use it with confidence.

"Hello, have you verified the effect of Wan's 14B full-parameter training using the training schedule provided in [Diffsynth](https://github.com/modelscope/DiffSynth-Studio) repository? Specifically, full parameters training with uniform sampling, timestep shift (shift = 5), and loss weighting?"

**Comment by Feynman1999** [comment]
> > [@RedRAINXXXX](https://github.com/RedRAINXXXX) Hello, I am a developer of DiffSynth-Studio.
> > Both of these approaches can be used for training. These differences also exist in the open-source training code of the FLUX model.
> > Our experimental results on FLUX indicate that such parameters perform the best in FLUX's LoRA training. Given the similarities between Wan and FLUX, we have adopted FLUX's experience in Wan's training.
> > We have already validated this approach in Wan's LoRA training, so you can use it with confidence.
> 
> "Hello, have you verified the effect of Wan's 14B full-parameter training using the training schedule provided in [Diffsynth](https://github.com/modelscope/DiffSynth-Studio) repository? Specifically, full parameters training with uniform sampling, timestep shift (shift = 5), and loss weighting?"

Can the script of DiffSynth-Studio be used to fully fine-tune the model of 14b? The gpu memory should not be enough? I understand that only LoRa training is possible
